stack

```diff
@@ -226,6 +245,7 @@ function resource-provider-docker-run() {
     --name resource-provider \
     --add-host localhost:host-gateway \
     --env-file <(doppler -p resource-provider -c dev secrets download --no-file --format docker) \
+    -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
If we are mounting the socket, can we remove the `--privileged` flag?

Reply:
we can! good catch. done :)